app/assets/javascripts/rentals/new.js.erb

```diff
@@ -69,4 +111,16 @@ $(document).ready( function () {
   $('select.cost-dependent').on('change', function () {
     costCalculation();
   });
+
+  // ~ user searching
```

Comment:
not at all, i was just commenting that the hooks below are all for user searching.